### DISCUSSION TITLE
can not open codespace for my account

### DISCUSSION THREAD

**Discussion by Faisal-Shumon** [community/community#35889] [Codespaces]
can not open codespace of my account. i use CS50  for Visual studio code for course.
plz help me.

Username- Faisal-Shumon

**Comment by Faisal-Shumon** [comment]
could anyone help me ?  from yesterday , i can not open  code50 codespace. just showing connecting ,  starting...... , but not open.

Browser console showing below msg:

Cookie “MS0” has been rejected for invalid domain
Cookie “MC1” has been rejected for invalid domain.

**Comment by jostasik** [comment]
Have codespaces worked for you in other repositories? I would first try it in a different browser, or an incognito window. 
Maybe a browser extension or some saved cookies are messing with it. That would narrow down the culprit to GitHub itself.

EDIT: i just saw that codespaces have had issues today, this might also be the cause? https://www.githubstatus.com/incidents/7hfyz8mjg663

**Reply by Faisal-Shumon** [reply]
i used visual studio code  for cs50 by my  GITHUB account.  i use Mozilla firefox for it. my browser is up to date.  i uninstalled/installed browser, but no improvement.
i am still getting prolem.
plz suggest me what shoud i do ?

**Comment by Faisal-Shumon** [comment]
can anyone help me, what should i do ? still can not open codespace for CS50 visual studio. 
just showing connecting or starting in the box   and loading loading .

**Comment by thomas-sickert** [comment]
Sorry to hear you're having trouble. Could you check the network tab in the browser developer tools for any errors there? To unblock yourself in the meantime, it's also worth trying to download Visual Studio Code and using codespace from the desktop application.
